pkg/k8s/util.go

```diff
@@ -0,0 +1,24 @@
+package k8s
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ExtensionAPIServerConfigMapResource returns the string representation of a ConfigMap resource, meant to be used in unit tests.
+func ExtensionAPIServerConfigMapResource(data map[string]string) string {
```

Comment:
I can see how this cleans up a bunch of duplication in the tests but I think I'd still prefer putting the manifest directly inline instead of a `map[string]string`.  The tests are going to be verbose anyway and I think it's helpful to see the manifest files.